test/typescript-validate.ts

```diff
@@ -1,4 +1,5 @@
-import * as Webhooks from "..";
+import { Webhooks } from "..";
+import { EventNames } from "../lib/generated/event-names";
```

Comment:
**[opinion❓]** Is a good approach to rely on `../lib/generated` or should we think on a structure of export { ... } inside ./index.d.ts to expose `EventNames` and `EventPayloads` from there?

Reply:
I think it could be a good idea to export them from the `index.d.ts`